qutebrowser/utils/usertypes.py

```diff
@@ -149,6 +171,13 @@ def _get_new_item(self, offset):
                 new = self.curitem()
             elif self._mode == self.Modes.exception:  # pragma: no branch
                 raise
+            elif self._mode == self.Modes.edge:
```

Comment:
Can you move the `# pragma: no branch` line from above to here? This is to tell `coverage.py` to not mark the (not existing) "else:" branch of this if as not taken, as there's no way this could happen.